src/common/cli_wrapper.cpp

```diff
@@ -132,6 +132,12 @@ void CLIWrapper::parseAliases() {
   if(aliases_.empty())
     return;
 
+  // Find the set of values allowed for each alias option.
+  // Later we will check and abort if an alias option has an unknown value.
+  std::unordered_map<std::string, std::unordered_set<std::string>> allowedAliasValues;
+  for(const auto &alias : aliases_)
```

Comment:
I prefer `auto&&` here to make sure it doesn't copy (but that's mostly an issue with 3rd party container that have poorly defined iterators... Doesn't make a difference here.

Reply:
Changed, thanks!